tutorial/03_segmentation_and_measuring.md

```diff
@@ -0,0 +1,726 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+%matplotlib inline
+%config InlineBackend.figure_format = 'retina'
+
+import numpy as np
+import scipy as sp
+import skimage as ski
+import matplotlib.pyplot as plt
+```
+
+# Part 3: Segmentation
+
+## Splitting an image into regions of interest
+
+--------------
+
+One can think of an image as a projection of a scene, consisting of multiple objects. Sometimes, the scene is considered as a whole, but quite often it is useful to identify the constituent parts, or segments. E.g., we may want to count those segments, or calculate some properties of them. E.g., the number and sizes of cells in a microscopy slide, or the positions and poses of people in a photograph.
+
+Splitting an image into its constituent parts is known as *segmentation*.
+
+Think of Michaelangelo, carving the statue of David. The block of marble is *segmented* into foreground and background. In this case, the background is discarded, and only the work of art remains.
+
+To be useful, segmentation has to be followed by *measurement* and *interpretation*. We measure attributes of the segments, and then interpret those measurements to ask some question. Ultimately, we want to know: are the cells healthy, or is the part manufactured correctly, or is the land flooded.
+
+```mermaid
+flowchart LR
+
+A[Imaging] --> B[Segmentation]
+B --> C[Measurement]
+C --> D[Interpretation]
+```
+
+Let's consider that second block, the segmentation algorithm, for a moment.
+
+Scikit-image implements several *heuristic* algorithms for segmentation. I.e., these are typically *unsupervised*, based on rules around properties of the image / pixels, rather than on labeling examples of objects and non-objects. Neural network approaches, such as U-Net, DeepLab, and Mask R-CNN have proved very effective for segmenting images, given enough training data. These are likely what would be used in practice, unless sufficient labeled data is unavailable.
```

Comment:
The double negative is twisting my brains... "unless not enough labeled data are available?"